lib/DDG/Rewrite.pm

```diff
@@ -156,8 +156,11 @@ sub _build_nginx_conf {
 
 	$cfg .= "\techo_before_body '".$self->callback."(';\n" if $wrap_jsonp_callback;
 	$cfg .= "\techo_before_body '".$self->callback.qq|("';\n| if $wrap_string_callback;
+	my ($spice_name) = $self->path =~ /js\/spice\/(.+?)\//;
```

Comment:
Does this (need to) work with [spices with child directories](https://github.com/duckduckgo/zeroclickinfo-spice/tree/master/share/spice/transit)?